cordova-lib/cordova-lib.js

```diff
@@ -18,19 +18,35 @@
 */
 
 // For now expose plugman and cordova just as they were in the old repos
+
+
+function addProperty(obj, property, modulePath) {
```

Comment:
Nice, that should definitely be consolidated. It will have to be moved into cordova-common though.